src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/AbstractImportCompletionProvider.cs

```diff
@@ -46,7 +46,7 @@ public override async Task ProvideCompletionsAsync(CompletionContext completionC
             var cancellationToken = completionContext.CancellationToken;
             var document = completionContext.Document;
 
-            var syntaxContext = await CreateContextAsync(document, completionContext.Position, cancellationToken).ConfigureAwait(false);
+            var syntaxContext = await completionContext.GetSyntaxContextWithExistingSpeculativeModelAsync(document, cancellationToken).ConfigureAwait(false);
 
```

Comment:
tbd.  not sure what the 'existing' part of the name means.

Reply:
Not sure why it uses this name, but it's the standard way to obtain a `SyntaxContext` from a `CompletionContext`.

Reply:
It came from `ReuseExistingSpeculativeModelAsync`